examples/fn_config/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"net/http"
+
+	fdk "github.com/CrowdStrike/foundry-fn-go"
+)
+
+func main() {
+	fdk.Run(context.Background(), newHandler)
+}
+
+// newHandler here is showing how a config is integrated. It is using generics,
+// so we can unmarshal the config into a concrete type and then validate it. The
+// OK method is run to validate the contents of the config.
+func newHandler(_ context.Context, cfg config) fdk.Handler {
+	mux := fdk.NewMux()
+	mux.Get("/foo", fdk.HandlerFn(func(ctx context.Context, r fdk.Request) fdk.Response {
+		return fdk.Response{
+			Body:   json.RawMessage(`{"foo":"val"}`),
+			Code:   200,
+			Header: http.Header{"X-Cs-Method": []string{r.Method}},
+		}
+	}))
+	mux.Post("/foo", fdk.HandlerFn(func(ctx context.Context, r fdk.Request) fdk.Response {
+		return fdk.Response{
+			Body:   r.Body,
+			Code:   201,
+			Header: http.Header{"X-Cs-Method": []string{r.Method}},
```

Comment:
Using an `X-CS-*` header here is going to be problematic if the request goes through the API Gateway because it filters out all `X-CS-*` headers.

Reply:
oiii fair point, I can remove that `X-Cs` prefix and make it `X-Fn-` for the examples